config/fcos/v1_5_exp/validate_test.go

```diff
@@ -317,15 +317,15 @@ func TestValidateConfig(t *testing.T) {
 				},
 			},
 			err:     common.ErrInvalidPartitionConfig,
-			errPath: path.New("yaml"),
+			errPath: path.New("partition_number", 0),
 		},
 	}
 
 	for i, test := range tests {
 		t.Run(fmt.Sprintf("validate %d", i), func(t *testing.T) {
 			actual := test.config.Validate()
 			expected := report.Report{}
-			expected.AddOnError(test.errPath, test.err)
+			expected.AddOnWarn(test.errPath, test.err)
 			assert.Equal(t, expected, actual, "invalid report")
 		})
 	}
```

Comment:
this seems to fix it, makes sense as the 0th partition is the one that causes the fail.

Reply:
Good job finding it!